src/database/database.ts

```diff
@@ -17,7 +20,10 @@ export const insertVtBooking = async function (): Promise<string[]> {
   vehicleBooking.FK_VEH_SYST_NO = 1234567;
   vehicleBooking.COUNTED_AXLES = 2;
 
-  const insertResult: string[] = await dbConnection()
+  const config: DatabaseConnectionConfig = process.env.ORACLE_CONFIG_SECRET
```

Comment:
This should be moved to dbConnection.ts. Here it will get called every time there's an insert.